### DISCUSSION TITLE
/switching-to-ios/

### DISCUSSION THREAD

**Discussion by migratecomments** [ironicbadger/blog.ktz.me#19] [Announcements]
Switching to iOS
        
[https://blog.ktz.me/switching-to-ios/](https://blog.ktz.me/switching-to-ios/)

**Comment by televisi** [comment]
Oh my, here I am in 2023 trying to go down the same path, trying Apple from Android; and I found a similar issue with it.

I've been using Macbook pro since 2010 for personal use (work always providing Windows laptop), thus relatively familiar with the Apple ecosystem; but lately decided to buy a second-hand iPad 8th gen, iPhone 12 Pro, and Apple Watch 7, in the hope that I can use the convenience closed-loop ecosystem that people are talking about. (note: I bought second hand, as I am not ready to spend $3K worth of devices)

I like the idea for you be able to use messages in iPad, iPhone, Macbook on the go; also the handoff feature.
But like you said, there is this quirky thing with iOS, you mentioned about 'comma', for me is the 'dot', where you need to change the keyboard to symbol to reach it (or you can use double space as a shortcut, but then it only shows a single 'dot', where sometimes you need to have multiple dots on the message...)

The other thing, I am a bilingual person, thus I speak other language, with English, I can swipe the words easily; with the other language, not so much...such a shame that Apple hasn't figured it out as yet, while Samsung has done it for a while.

And yes, I missed nzb360 too

Honestly, I'm still on the fence now whether to sell my iPhone and Apple watch; I still keep my Galaxy S23 and Galaxy Watch S5, 'just in case', if you know what I mean... I guess time will tell....
